frontend/src/css/FeedItemWithPreview.styl

```diff
@@ -28,7 +28,12 @@ smallScreenFeedWidth = 400px
     &__texteditor
       margin 0
     &__messagelist
-      padding 0
+        padding 0
```

Comment:
is the spacing correct here?

Reply:
Good catch. It'd be useful to have a linter for stylus files.

Reply:
fixed